examples/uros_6lowpan/uros_6lowpan_main.c

```diff
@@ -0,0 +1,197 @@
+#include <nuttx/config.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+//uROS libraries
+#include <rcl/rcl.h>
+#include <rcl/error_handling.h>
+#include <std_msgs/msg/int32.h>
+#include <rmw_uros/options.h>
+
+#if defined(BUILD_MODULE)
+int main(int argc, char *argv[])
+#else
+int uros_6lowpan_main(int argc, char* argv[])
+#endif
+{
+    char buffer[256]; // Buffer to save configuration commands.
+    char aux_buffer[2]; // Buffer to save the configuration ID.
```

Comment:
Must be length 3 because of trailing \0
```suggestion
    char aux_buffer[3]; // Buffer to save the configuration ID.
```